src/site/content/en/blog/contact-picker/index.md

```diff
@@ -0,0 +1,321 @@
+
+Access to the users' contacts is via the picker, and it can only be called with
+a user gesture, on a [secure][secure-contexts], top-level browsing context.
+This ensures that a site can't show the picker on page load, or randomly show
+the picker without any context.
+
+<figure class="w-figure w-figure--inline-right">
+  <img src="contact-picker-user-choice.jpg" class="w-screenshot"
+       alt="Screen shot, users can choose which properties to share.">
+  <figcaption>
+    Users can choose not to share some properties. In this screenshot, the
+    user has unchecked the 'Phone numbers' button. Even though the site
+    asked for phone numbers, they will not be shared with the site.
+  </figcaption>
+</figure>
+
+There's no option to bulk-select all contacts so that users are encouraged
+to select only the contacts that they need to share for that particular
+website. Users can also control which properties are shared with the site
+by toggling the property button at the top of the picker.
+
+<div class="clearfix"></div>
+
+### Transparency {: #security-transparency }
+
+To clarify which contact details are being shared, the picker always
+shows the contact's name and icon, plus any properties that the site has
+requested. For example, if a site requests `name`, `email`, and `tel`,
+all three properties will be shown in the picker. Alternatively,
+if a site only requests `tel`, the picker will show only the name, and
+telephone numbers.
+
+<figure class="w-figure w-figure--inline-left" style="max-width:42%">
+  <img src="contact-picker-left.jpg" class="w-screenshot"
+       alt="Screen shot of picker for site requesting all properties.">
+  <figcaption>
+    Picker, site requesting <code>name</code>, <code>email</code>, and
+    <code>tel</code>, one contact selected.
+  </figcaption>
+</figure>
+<figure class="w-figure w-figure--inline-right" style="max-width:42%">
+  <img src="contact-picker-right.jpg" class="w-screenshot"
+       alt="Screen shot of picker for site requesting only phone numbers.">
+  <figcaption>
+    Picker, site requesting only <code>tel</code>, one contact selected.
+  </figcaption>
+</figure>
+
+<div class="clearfix"></div>
+
+<figure class="w-figure w-figure--inline-right">
+  <img src="contact-picker-long-press.jpg" class="w-screenshot"
+       alt="Screen shot of picker when a contact is long-pressed.">
+  <figcaption>
+    The result of a long press on a contact.
+  </figcaption>
+</figure>
+
+A long press on a contact will show all of the information that will be
+shared if the contact is selected. (See the Cheshire Cat contact image.)
+
+<div class="clearfix"></div>
+
+### No permission persistence {: #security-persistence }
+
+Access to contacts is on-demand, and not persisted. Each time a site wants
+access, it must call `navigator.contacts.select()` with a user gesture,
+and the user must individually choose the contact(s) they want to share
+with the site.
+
+## Feedback {: #feedback }
```

Comment:
Something to think about if you want to iterate on these posts...

Michael made a nice Details component to support the PWA checklist.
https://deploy-preview-1556--web-dev-staging.netlify.com/handbook/web-dev-components/#details

It might look neat to try to put these sections into Details components to see if they render as a kind of stacked list. I'm not sure if it would look good or bad, but wanted to mention it as a possibility 😁

Reply:
It is pretty, but I don't want to hide the content and make a user click to open it.